themes/nekonight/nekonight.theme.sh

```diff
@@ -0,0 +1,42 @@
+# nekonight Bash prompt with source control management
+# Theme inspired by:
+#  - Bash_it cupcake theme
+# Demo:
+# ╭─🐱 virtualenv 🐱user at 🐱host in 🐱directory on (🐱branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗)
+# ╰λ cd ~/path/to/your-directory
+
+icon_start="╭─"
+icon_user=" 🐱 \[\e[1;33m\]\u\[\e[0m\]" # Yellow-colored username
```

Comment:
These color sequences are supposed to be specified through `$_omb_prompt_bold_olive`, `$_omb_prompt_normal`, etc. Please check `lib/omb-prompt-colors.sh` for definitions or other themes for reference.